go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/aws-controllers-k8s/iam-controller
 go 1.14
 
 require (
-	github.com/aws-controllers-k8s/runtime v0.14.1
+	github.com/aws-controllers-k8s/runtime v0.15.2
 	github.com/aws/aws-sdk-go v1.37.10
```

Comment:
Maybe generate with the most recent version or `v1.42`

Reply:
done.